nms/packages/magmalte/app/views/subscriber/SubscriberTable.js

```diff
@@ -194,75 +194,124 @@ const StyledMenu = withStyles({
   />
 ));
 
+export type SubscriberActionType = 'add' | 'edit' | 'delete';
+
 function SubscriberActionsMenu(props: {onClose: () => void}) {
   const [anchorEl, setAnchorEl] = React.useState(null);
   const [open, setOpen] = React.useState(false);
   const [error, setError] = React.useState('');
   const enqueueSnackbar = useEnqueueSnackbar();
   const ctx = useContext(SubscriberContext);
   const successCountRef = useRef(0);
+  const [
+    subscriberAction,
+    setSubscriberAction,
+  ] = useState<SubscriberActionType>('add');
   const handleClick = event => {
     setAnchorEl(event.currentTarget);
   };
 
   const handleClose = () => {
     setAnchorEl(null);
   };
+
   /**
-   * Post subscriber chunk or update subscriber(s) if param edit is set to true
+   * Delete Subscribers using a csv file or selecting subscribers with a checkbox in subscriber delete dialog.
```

Comment:
Just a nit about this description - this function doesn't actually receive a csv file or have you select the checkboxes, that's the actual component itself, which passes the results of the CSV or the checkboxes to this. I think we should be careful to be very clear in these function signatures and descriptions, because that's all that separates a lot of these very similar looking functions.